lib/effective_datatables.rb

```diff
@@ -47,4 +49,17 @@ def self.find(id)
     klass.try(:new) || raise('unable to find datatable')
   end
 
+  # Locale is coming from view. I think it can be dynamic.
+  # We currently support: en, es, nl
+  def self.language(locale)
+    @_languages ||= {}
+
+    locale = :en unless AVAILABLE_LOCALES.include?(locale.to_s)
+
+    @_languages[locale] ||= begin
+      path = Gem::Specification.find_by_name('effective_datatables').gem_dir + "/app/assets/javascripts/dataTables/locales/#{locale}.lang"
+      JSON.parse(File.read(path)).to_json
+    end
+  end
```

Comment:
I just wanted to memoize the json here, to avoid reading a file every time a datatable is rendered.  I think a small memory overhead is a good trade off here.